### DISCUSSION TITLE
Is there a macOS version?

### DISCUSSION THREAD

**Discussion by FredoMarr** [porrey/Virtual-ZPL-Printer#9] [Ideas]
Is there a way to use this in a Mac environment? or any plans on making a MacOS version?

**Comment by porrey** [comment]
I am not aware of any way to use this currently on a Mac because it is heavily Windows dependent (it is a WPF). 

I do not have a timeline but my thought was to rebuild it using MAUI so it was cross platform. I am a Mac person myself and actually do my development on a MacBook Pro.

**Reply by FredoMarr** [reply]
Ah gotcha, thanks for responding so quickly.
It definitely would be nice to have a macOS version and MAUI would be a great update for this kind of app, especially for cross-platform development.

Please update us when you do make it!
